awx/settings/production.py

```diff
@@ -44,7 +44,7 @@
 SCHEDULE_METADATA_LOCATION = '/var/lib/awx/.tower_cycle'
 
 # Ansible base virtualenv paths and enablement
-BASE_VENV_PATH = "/var/lib/awx/venv"
+BASE_VENV_PATH = os.path.realpath("/var/lib/awx/venv")
```

Comment:
`bwrap` won't follow symlinks

Reply:
why here instead of where it's used later?

Reply:
<img src="https://i.ytimg.com/vi/oWCcB_fOOLc/maxresdefault.jpg" />

Reply:
But really, there's a few other places where this setting is consulted (like in the code where we look for custom venvs).

Also, this patch comes from the field; @gamuniz had a user complaining about this issue apply _this_ patch to `production.py` and it resolved their issue.

Reply:
yep safest place is top level, unless you're saying we should change it everywhere? The user I helped with this setting has reported it fixed their issue

Reply:
Just saying, this only works if /var/lib/awx or /var/lib/awx/venv is a symlink. If there are symlinks *in* that dir, it will still fail.

Reply:
ahh. Yeah i think if the use-case is to use another directory for venvs we could use the custom_venv_paths, this fix is for those who have their home directory symlinked